erigon-lib/state/domain_shared.go

```diff
@@ -1027,23 +1027,22 @@ func (sdc *SharedDomainsCommitmentContext) Account(plainKey []byte) (u *commitme
 	u.Flags |= commitment.BalanceUpdate
```

Comment:
I don't know why we consider this to be a balance update? What if the balance wasn't updated?

Reply:
how you gonna check? asking history is not worth the effort, easier to mark balance as updated anyways, so balance will not be ignored in the middle of something. Account() happens only for keys we really know need to load (no memoized hash | key was directly given through `Updates` list as being touched). Balance is used along with Nonce during hashing so only makes sense to load them both or neither.